core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.html

```diff
@@ -447,25 +447,6 @@ <h4 class="bold-title">
                 </div>
             </section>
             <!-- end of aws events -->
-            <section class="each-step-wrap cloudfront-url" *ngIf="typeOfService == 'website'">
```

Comment:
I understand that this  feature is not enabled/used at this point, but let's keep it in here so that we can enable it if we need at a later point of time.